src/watchdog.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+'''
+xxx check number of days until certificate expires
+xxx check number of pending software updates
+'''
+
+import datetime
+import glob
+import os
+import random
+import sys
+
+import config
+import database
+import entities
+import geoip
+import mail
+
+CONFIG_FILENAME = '/srv/web/app-config.json'
```

Comment:
This probably shouldn't be hardcoded

Reply:
yeah, you're right. I'll make the config file a command-line argument and get all the other paths and filenames from there.

Reply:
... in a future PR.